assembly/__tests__/sentences.ts

```diff
@@ -144,3 +144,7 @@ class ReverseArgs {
   ) { }
 }
 
```

Comment:
I would add a exported function here that does contract patch actions.  Then you can use the `runtimes/__tests__/cross.spec.ts` to test it.

Reply:
added